src/libraries/System.Security.Cryptography.Encoding/tests/Resources/Strings.resx

```diff
@@ -1,4 +1,64 @@
-﻿<root>
+﻿<?xml version="1.0" encoding="utf-8"?>
+<root>
+  <!-- 
```

Comment:
This was added by the tooling. Any reason why I shouldn't keep it?

Reply:
I think we've removed it from everywhere else.